Dotnet-Install.ps1

```diff
@@ -429,6 +433,16 @@ function Prepend-Sdk-InstallRoot-To-Path([string]$InstallRoot, [string]$BinFolde
     }
 }
 
+function Disable-MultiLevelLookup() {
+    if (-Not $NoDisableMultiLevelLookup) {
+        Say "Adding to current process DOTNET_MULTILEVEL_LOOKUP: `"0`". Note: This change will not be visible if PowerShell was run as a child process."
+        $env:DOTNET_MULTILEVEL_LOOKUP = "0"
+    }
+    else {
+        Say "MultiLevelLookup has NOT been disabled. The dotnet host may resolve dependencies from the machine global dotnet install location"
+    }
+}
+
```

Comment:
Is the purpose of putting this here just for the side-effect? or does it affect the behavior of dotnet-install itself? 

If the former, it should be in the createstore script. The CreateStore script already sets some environment variables and is basically required to run already.

Reply:
Its just for the side-effect. Putting it in CreateStore wouldn't be my preference though I don't mind if that is what everyone else wants. Often I don't use CreateStore (dotnet installs include their own store by default now) because I am not making changes that affect the images. However if consensus says move it to CreateStore I'll just set it myself manually at the command line.